app/lib/katello/resources/candlepin/consumer.rb

```diff
@@ -92,7 +93,7 @@ def checkin(uuid, checkin_date)
             self.put(path(uuid), {:lastCheckin => checkin_date}.to_json, self.default_headers).body
           end
 
-          def available_pools(owner_label, uuid, listall = false)
+          def available_pools(owner_label, uuid, listall: false)
```

Comment:
Why was this changed?

Reply:
I changed a lot of methods to use keyword arguments because it's in Katello's Rubocop config. Turns out, CI doesn't care because it uses Foreman's Rubocop config. But I figured it wouldn't hurt anything to leave the fixes (it's more modern Ruby anyway.)